test/export/test_torchbind.py

```diff
@@ -392,6 +403,52 @@ def forward(self, arg0_1, attr, arg1_1):
     return (getitem_3, add_1)""",  # noqa: B950
         )
 
+    def test_make_fx_tensor_queue_methods(self):
+        class Model(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.linear = torch.nn.Linear(3, 2)
+
+            def forward(self, tq, x):
+                tq.push(x.cos())
```

Comment:
Can you add an `isinstance(tq, ???)` check? 

What is tq? is it a FakeTensorQueue or an AbstractScriptObject?

Reply:
Yeah, I can add the check here. It is AbstractScriptObject.